stratum/hal/lib/dummy/dummy_phal.cc

```diff
@@ -25,6 +25,11 @@
 #include "stratum/glue/logging.h"
```

Comment:
no need to add change to `dummy_phal` since the dummy switch is using the **generic phal** now

Reply:
Removed TAI related code from dummy phal. Please review.